modulefiles/build_noaacloud_intel.lua

```diff
@@ -5,13 +5,13 @@ the NOAA cloud using Intel-oneapi
 
 whatis([===[Loads libraries needed for building the UFS SRW App on NOAA cloud ]===])
 
-prepend_path("MODULEPATH", "/contrib/EPIC/spack-stack/envs/srw-develop-intel/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/contrib/EPIC/spack-stack/spack-stack-1.4.1/envs/unified-dev/install/modulefiles/Core")
 prepend_path("MODULEPATH", "/apps/modules/modulefiles")
 prepend_path("PATH", "/contrib/EPIC/bin")
-load("intel/2021.3.0")
-load("impi/2021.3.0")
+--load("intel/2021.3.0")
+--load("impi/2021.3.0")
```

Comment:
Will these commented out lines be used in the future?  If not, they should be removed to clean up the modulefile.

Reply:
removed.

Reply:
Thanks!